visual-js/visual-storybook/integration-tests/tsconfig.json

```diff
@@ -21,6 +21,7 @@
     "jsx": "react-jsx"
   },
   "include": [
-    "src"
+    "src/**/*.ts",
```

Comment:
Shouldn't need to specify file types here `src/**/*` should be enough and it will know to choose ts/tsx.